libraries/models.py

```diff
@@ -40,6 +40,17 @@ def get_sample(self, is_bot=False):
     def ratio_followers_following_per_users(self, users):
         return [user.ratio_followers_following() for user in users]
 
+    @classmethod
+    def followers_friends_per_users(self, users):
+        follow_df = DataFrame(columns=["followers","following","accountreputation","CDFx","CDFy"], index=range(len(users)))
+        follow_df["followers"] = [user.followers for user in users]
```

Comment:
je sais que tu aimes cette notation, mais la tu fais 3 fois la meme boucle `for user in users`